src/overlays/actors/ovl_En_Tr/z_en_tr.c

```diff
@@ -19,9 +19,9 @@ void EnTr_ShrinkVanish(EnTr* this, PlayState* play);
 void EnTr_WaitToReappear(EnTr* this, PlayState* play);
 void EnTr_ChooseAction1(EnTr* this, PlayState* play);
 
-void EnTr_UpdateRotation(EnTr* this, PlayState* play, s32 actionIndex);
-void func_80B24038(EnTr* this, PlayState* play, s32 actionIndex);
-void EnTr_SetStartPosRot(EnTr* this, PlayState* play, s32 actionIndex);
+void EnTr_SetCueRot(EnTr* this, PlayState* play, s32 cueChannel);
+void func_80B24038(EnTr* this, PlayState* play, s32 cueChannel);
+void EnTr_SetCueStartPosRot(EnTr* this, PlayState* play, s32 cueChannel);
```

Comment:
I'm not a fan of the `SetCueXxx` names, it sounds like it would write to the `CsCmdActorCue` structs

a suggestion could be `SetXxxFromCue` maybe?